actor/v2action/service_access_test.go

```diff
@@ -495,6 +513,29 @@ var _ = Describe("Service Access", func() {
 				Expect(orgGUID2).To(Equal("org-guid-1"))
 				Expect([]string{planGUID1, planGUID2}).To(ConsistOf([]string{"service-plan-guid-1", "service-plan-guid-2"}))
 			})
+
+			When("service is already enabled for all orgs for a plan", func() {
```

Comment:
This `When` and accompanying `It` have odd descriptions. For one, we're only dealing with a single org here; for two, the test itself shows that a plan visibility *is* being created - for the plan that isn't public. 

Please update these descriptions to more accurately reflect what is being tested.